proto-app/src/containers/RequestHistory/RequestHistory.jsx

```diff
@@ -1,22 +1,334 @@
+		fileId: "44444444 - 4444 - 4444 - 4444 - 44444444444",
+		type: "DOCX",
+		name: "test.docx",
+		outcome: "Blocked",
+		url: "https://www.test-site.com/test.docx",
+		host: "1.1.1.1",
+		clientAddress: "192.168.1.1",
+		proxyAddress: "192.168.2.2",
+		report: "https://www.test-site.com/report",
+		reqTime: "3000ms",
+		rebTime: "2500ms",
+	},
+	{
+		id: "file-05",
+		timestamp: new Date().toUTCString(),
+		fileId: "55555555 - 5555 - 5555 - 5555 - 55555555555",
+		type: "PNG",
+		name: "test.png",
+		outcome: "Safe",
+		url: "https://www.test-site.com/test.png",
+		host: "1.1.1.1",
+		clientAddress: "192.168.1.1",
+		proxyAddress: "192.168.2.2",
+		report: "https://www.test-site.com/report",
+		reqTime: "3000ms",
+		rebTime: "2500ms",
+	},
+	{
+		id: "file-06",
+		timestamp: new Date().toUTCString(),
+		fileId: "66666666 - 6666 - 6666 - 6666 - 66666666666",
+		type: "DOCX",
+		name: "test.docx",
+		outcome: "Blocked",
+		url: "https://www.test-site.com/test.docx",
+		host: "1.1.1.1",
+		clientAddress: "192.168.1.1",
+		proxyAddress: "192.168.2.2",
+		report: "https://www.test-site.com/report",
+		reqTime: "3000ms",
+		rebTime: "2500ms",
+	},
+	{
+		id: "file-07",
+		timestamp: new Date().toUTCString(),
+		fileId: "77777777 - 7777 - 7777 - 7777 - 77777777777",
+		type: "PNG",
+		name: "test.png",
+		outcome: "Safe",
+		url: "https://www.test-site.com/test.png",
+		host: "1.1.1.1",
+		clientAddress: "192.168.1.1",
+		proxyAddress: "192.168.2.2",
+		report: "https://www.test-site.com/report",
+		reqTime: "3000ms",
+		rebTime: "2500ms",
+	},
+	{
+		id: "file-08",
+		timestamp: new Date().toUTCString(),
+		fileId: "88888888 - 8888 - 8888 - 8888 - 88888888888",
+		type: "DOCX",
+		name: "test.docx",
+		outcome: "Blocked",
+		url: "https://www.test-site.com/test.docx",
+		host: "1.1.1.1",
+		clientAddress: "192.168.1.1",
+		proxyAddress: "192.168.2.2",
+		report: "https://www.test-site.com/report",
+		reqTime: "3000ms",
+		rebTime: "2500ms",
+	},
+	{
+		id: "file-09",
+		timestamp: new Date().toUTCString(),
+		fileId: "99999999 - 9999 - 9999 - 9999 - 99999999999",
+		type: "PNG",
+		name: "test.png",
+		outcome: "Safe",
+		url: "https://www.test-site.com/test.png",
+		host: "1.1.1.1",
+		clientAddress: "192.168.1.1",
+		proxyAddress: "192.168.2.2",
+		report: "https://www.test-site.com/report",
+		reqTime: "3000ms",
+		rebTime: "2500ms",
+	},
+	{
+		id: "file-10",
+		timestamp: new Date().toUTCString(),
+		fileId: "10101010 - 1010 - 1010 - 1010 - 10101010101",
+		type: "DOCX",
+		name: "test.docx",
+		outcome: "Blocked",
+		url: "https://www.test-site.com/test.docx",
+		host: "1.1.1.1",
+		clientAddress: "192.168.1.1",
+		proxyAddress: "192.168.2.2",
+		report: "https://www.test-site.com/report",
+		reqTime: "3000ms",
+		rebTime: "2500ms",
+	},
+];
+
+const fileTypeList = [
+	{
+		id: "microsoftword",
+		filterName: "Microsoft Word",
+		checkboxList: [
+			{ type: "radio", head: "doc", name: "docType", id: "doc" },
```

Comment:
```suggestion
			{ type: "checkbox", head: "doc", name: "docType", id: "doc" },
```


need to change all of these to checkboxes, you should be able to view more than one file type in each group